mappings/net/minecraft/entity/passive/TadpoleEntity.mapping

```diff
@@ -0,0 +1,21 @@
+CLASS net/minecraft/unmapped/C_gdxvnoho net/minecraft/entity/passive/TadpoleEntity
+	FIELD f_epyeeylc tadpoleAge I
+	FIELD f_jetulgup GROWTH_AGE I
+	FIELD f_lhkulahc MODULES Lcom/google/common/collect/ImmutableList;
+	FIELD f_rfntsubn SENSORS Lcom/google/common/collect/ImmutableList;
+	METHOD m_dsreleic decrementStack (Lnet/minecraft/unmapped/C_jzrpycqo;Lnet/minecraft/unmapped/C_sddaxwyk;)V
+		ARG 1 player
+		ARG 2 stack
+	METHOD m_exzbgvfj testIsFood (Lnet/minecraft/unmapped/C_sddaxwyk;)Z
+		ARG 1 stack
+	METHOD m_fpexwmxb setAge (I)V
+		ARG 1 age
+	METHOD m_iieprzwi growSeconds (I)V
+		ARG 1 seconds
+	METHOD m_jlrrvacd getTadpoleAge ()I
+	METHOD m_nktwhaku timeTillGrow ()I
+	METHOD m_owndoqec createTadpoleAttributes ()Lnet/minecraft/unmapped/C_sdjuuzrz$C_tehwrjus;
+	METHOD m_pkfuobvm growIntoFrog ()V
```

Comment:
This is horrible to imagine, just morphs into a frog. this can be kept but turnIntoFrog also works. turnToFrog works too.

Reply:
could you elablorate?

Reply:
growIntoFrog seems good to me

Reply:
turnIntoFrog but yeah this also works it is just hilarious to imagine my previous comment